tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -65,213 +65,212 @@ def header_colors():
-                                        value=1,
-                                        type='number',
-                                    )
-                                ],
-                            ),
-                            html.Div(
-                                className='vp-vertical-style',
-                                title='Move horizontal dashed line',
-                                children=[
-                                    html.Div(
-                                        "Threshold",
-                                        className='vp-text',
-                                    ),
-                                    daq.Slider(
-                                        className='vp-input',
-                                        id='vp-genomic-line',
-                                        value=4,
-                                        max=10,
-                                        min=0,
-                                        handleLabel={'showCurrentValue': True, 'label': ' '},
-                                        step=0.01,
-                                        marks={str(num): str(num) for num in range(0, 11, 2)}
-
-                                    ),
-                                    dcc.Input(
-                                        className='vp-test-util-div',
-                                        id='vp-genomic-line-val',
-                                        value=4,
-                                        type='number',
-                                    )
-                                ],
-                            ),
-                        ],
-                    ),
-                    html.Div(
-                        id='vp-indicators-div',
-                        children=[
+    return html.Div(id='vp-page-content', children=[
+        html.Div(
+            id='vp-graph-div',
+            children=dcc.Graph(
+                id='vp-graph'
+            ),
+        ),
+        html.Div(id='vp-control-tabs', children=[
+            dcc.Tabs(id='vp-tabs', value='what-is', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='vp-tab', children=[
+                        html.H4('What is Volcano Plot?'),
+                        html.P(
+                            'You can use volcano plot to interactively '
+                            'identify clinically meaningful markers in '
+                            'genomic experiments, i.e., markers that are '
+                            'statistically significant and have an effect '
+                            'size greater than some threshold. '
+                            'Specifically, volcano plots depict the negative '
+                            'log-base-10 p-values plotted against their '
+                            'effect size, odds ratio, or log fold-change. '
```

Comment:
```suggestion
                            'effect size.'
```
I know this is not the focus of the PR, but I would like to take this opportunity to fix the description: Volcano plots may conventionally be used to plot ... against one of "effect size, odds ratio, or log fold-change" but, in the current version of this demo app, I can't find any way to change the x-axis to represent odds ratio or log fold-change, so I find it deceiving to mention them in this context.